cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIHDFSSink.java

```diff
@@ -1114,7 +1114,7 @@ private void persistMDAggregations(HDFSAggregator aggregator) throws CygnusPersi
                     
                     break;
                 } catch (CygnusPersistenceError | CygnusRuntimeError e) {
-                    LOGGER.info("[" + this.getName() + "] There was some problem with the current endpoint, "
+                    LOGGER.error("[" + this.getName() + "] There was some problem with the current endpoint, "
```

Comment:
I understand this change gets covered by this entry in ChangeLog:

```
[cygnus-ngsi] Changed log level for some events (#1768)
```

In that case, it should be modified to include the issue addressed by this PR, i.e.:

```
[cygnus-ngsi] Changed log level for some events (#1768, #1769)
```

Reply:
Fixed in [1fafe6c](https://github.com/telefonicaid/fiware-cygnus/pull/1774/commits/1fafe6cbeea5973d67b18197a7c47abb26493a6f)